### PR TITLE
Aria Huang HW01

### PR DESCRIPTION
**Pull request by anariaah**
In this assignment, I followed the instructions to complete a self-introduction using R studio with the incorporation of R markdown and Markdown language. I firstly read the readings on workflow to understand the basic coding process in R, and read the Markdown website, where I can learn about syntax and basic. I used different layers of headers, phrase emphasis, lists, links, and images. I also searched about the R Markdown website the found a cheatsheet on it. 

One of the challenge is to incorporate image. At first, I did not put my image in the same folder as my markdown file and did not upload it to the GitHub. When I run the code, no image displayed on the website. I fixed this problem and everything went well. 

One feedback is that R markdown is useful to formatting, typesetting, and layout design, especially for academic reports.

### CODE REVIEW

**Comment by anariaah** [comment]
I realized that I did not put my comments or reflections in the description! I actually put them in my codes, using <--Text comment-->

**Comment by anariaah** [comment]
I realized that I should put my photo in the same folder with the markdown file, so I can insert image in the local laptop instead of inserting a URL. So I made changes.

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
+install.packages("rmarkdown")
```
Comment (deblnia) [inline README.md:1]: I'd take this out -- you can't run R code in plain markdown!

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+---
+title: "self introduction""
+author: "Aria""
```
Comment (deblnia) [inline README.md:4]: Same with YAML (that's particular to RMarkdown)

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+  2. baking.
+  3. painting.
+My favorite show is [Reply 1988](https://en.wikipedia.org/wiki/Reply_1988).
+
+#### Image
+Here is my image
+  ![](/ariawithcoco.jpg).
+
+<!--My reflection on this assignment is that I firstly did not open a markdown files, so I could not run the Rmarkdown function. Then I went to read the RMarkdown instruction and the Markdown website to look into those syntax. I also found the cheetsheet for RMarkdown-->
+<!-- Another problem is that I cannot see my photo on GitHub. Then I change the path for my photo from my local computer to a facebook photo link. It then successfully displayed on the website-->
```
Comment (deblnia) [inline README.md:26]: You can't see it but your picture is there!

File: README.md.R [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
+==========
```
Comment (deblnia) [inline README.md.R:2]: What is this?

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     X         |            |
| **Coding strategy**         |           |              |          X  |
| **Presentation**            |           |    X          |            |
| **Achievement, creativity** |           |    X          |            |
| **Reproducibility**         |           |      X        |            |

Remarks:

* It seems like you're comfortable with the git submission workflow and you've used a bunch of markdown syntax -- both are great! 
* You seem to have some confusion about file types, so let me see if I can clear that up. Your README has the extension `.md` which stands for plain old markdown (because README's are kind of specific to Github, it's called Github flavored markdown). You can't run R code in this type of file. 
* You can also have a R script -- file extension `.R` -- (which is has R code that you can run). You can't use markdown syntax in there. 
* Photos are somewhat big files, so Github does not render them. You can see your photo on Github [here](https://github.com/anariaah/hw01/blob/master/ariawithcoco.jpg), or you can use a relative link as you did in your README.

**Review by elwebster — COMMENTED** [review]
Hi Aria! It was great to get to know you a bit. Your research sounds really interesting! I really liked your use of two different kinds of lists to denote research and personal interests (also, I'll have to check that show out, thanks for including the link). The use of italics to draw the eye to research interests was really well placed, as well. This was already mentioned by Deblina, but the R code does show up in the markdown file. I don't know if there's a way to solve this other than deleting the code in a basic markdown file, but if you're using r-markdown a good way to solve this if you want to keep the code for your own tracking is to use ```{r echo = FALSE} (code) (then hit enter and add three more grave accents when the code chunk is finished)

It would have been great to have a bit more context for the image - what's your stuffed friend's name? What is their significance to you? On the topic of the image, it seems like you did a great job solving this problem. You mention in your comments that you linked to a facebook website, but it looks like you also added the image to the repo directly. I would be curious to hear a bit more about whether the facebook link worked for you, as I similarly had to remove a local file pathway and add the image to the repo instead.

Just one more style note - the small-text title and author above the secondary HW01 title are a little visually confusing - you might consider specifying fontsize in R markdown (you can do this underneath title and author) or having these set as headers with HW01 as a subheader. Just a thought! 

Thanks for sharing!

**Pull request closed without merging**